### PR TITLE
Api features

### PR DESCRIPTION
**Pull request by m15e**
- [x] app fetches weather data from api upon user inputing location

- [x] app has switch to toggle between F/C temperature units

- [x] uses webpack

- [x] app meets ODIN project requirements

### CODE REVIEW

**Comment by m15e** [comment]
NOTE TO TSE: remote linter jobs not working for some reason, locally all tests are passing, any advice on how to fix this would be appreciated

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @m15e  👋

**Good work here. 👌**

- [ ] Kindly check the comments on your code for changes.

- [ ] OPTIONAL - Consider adding a loading icon to display while retrieving info.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "weather-app",
+  "version": "0.0.1",
+  "description": "weather app to play with OpenWeatherMap API",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```
Comment (mikenath223) [inline package.json]: - [x] Kindly include the necessary scripts for webpack.
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

**Review by bellom — APPROVED** [review]
## Hi @m15e  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_*

**Pull request merged**